File: backend/src/main/java/com/example/shoestore/dto/request/PlaceOrderRequest.java
package com.example.shoestore.dto.request;

import com.example.shoestore.enums.DeliveryMethod;
import com.example.shoestore.enums.PaymentMethod;
import lombok.*;

import java.math.BigDecimal;
import java.util.List;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class PlaceOrderRequest {
    private Integer userId;
    private List<Integer> selectedCartItemIds;
    private String voucherCode;
    private PaymentMethod paymentMethod;
    private DeliveryMethod deliveryMethod;
}

File: android/app/src/main/java/com/example/shoeshop/dto/request/PlaceOrderRequest.kt
package com.example.shoeshop.dto.request

import com.example.shoeshop.enums.DeliveryMethod
import com.example.shoeshop.enums.PaymentMethod
data class PlaceOrderRequest(
    val userId: Int,
    val selectedCartItemIds: List<Int>,
    val voucherCode: String,
    val paymentMethod: PaymentMethod,
    val deliveryMethod: DeliveryMethod
)
